src/priority_queue.c

```diff
@@ -74,7 +74,7 @@ PriorityQueue pq_create(ptrdiff_t max_size, ptrdiff_t *heap, ptrdiff_t *back,
 
   if (flag) {
     q.count = max_size;
-    build_heap(&q);
+    pq_deletemin(&q);
   }
```

Comment:
This change also causes a test failure for our `excesstopography_fmm*` functions, presumably because their priority queues are not constructed fully. Reverting it makes all our existing tests pass.

Reply:
Ah no, I did not notice I changed that, I initially used your implementation of priority queue and changed to mine because I needed specific operations. So it must be a mistake

Reply:
So just to be clear that was not intentional and you can reverse any change I made to any file not in `graphflood` subfolder (apart from `topotoolbox.h` or `CMakeLists.txt`)